spotify-conversion-next/services/spotify/spotifyRoutes.ts

```diff
@@ -0,0 +1,10 @@
+// This file is for condensing all of the URLs related to spotify into one spot so we just have to change them in one spot if they're ever used more than once!
+
+export const spotifyRoutes = {
+  // redirectLogin: `https://accounts.spotify.com/authorize?response_type=code&client_id=${process.env.SPOTIFY_ID}&scope=user-read-private user-read-email&redirect_uri=${process.env.SPOTIFY_REDIRECT_URI}`,

```

Comment:
the `redirectLogin` and `redirectLogout` URLs are just... default signin and signout URLs from the `next-auth` package. I'm telling ya, the package is just nuts.